### PR TITLE
Fixes 4217: Markdown is fixed for yaydoc's need

### LINKED ISSUE #4217: Fix markdown for yaydoc documentation generation
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-orga-server

**description**
Yaydoc needs markdown in a specific format so, alter the markdown according to the Yaydoc's documentation

### PR DESCRIPTION
**Pull request by sch00lb0y**
Fixes #4217 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `nextgen` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
I have made changes to  the markdown which need by the yaydoc to generate documentation
generated documentation: http://yaydoc.herokuapp.com/preview/rbalajis25@gmail.com/11e8c332-6e57-49f6-8821-88fe7c948e9a_preview/
#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by sch00lb0y** [comment]
@pri22296 @imujjwal96 please review.

File: README.md [hunk 1]
```diff
@@ -33,15 +33,15 @@ A demo version is automatically deployed from our repositories:
...
-1. [Generic Installation Instructions](/docs/installation/basic.md)
-1. [Local Installation](/docs/installation/local.md)
-1. [Vagrant Installation](/docs/installation/vagrant.md)
-1. [Deployment on Google Compute Engine](/docs/installation/google.md)
-1. [Deployment on Google Container Engine (Kubernetes)](/docs/installation/gce-kubernetes.md)
-1. [Deployment on AWS EC2](/docs/installation/aws.md)
-1. [Deployment on Digital Ocean](/docs/installation/digital-ocean.md)
-1. [Deployment with Docker](/docs/installation/docker.md)
-1. [Deployment on Heroku](/docs/installation/heroku.md)
+1. [Generic Installation Instructions](/docs/Installation/basic.md)
```
Comment (pri22296) [inline README.md]: why this change?

Comment (sch00lb0y) [inline README.md]: While documentation rendering 
bulleted point coming in small letter `installation` as folder name so I renamed the folder name as `Installation` that why I have changed all the references

Comment (pri22296) [inline README.md]: Can you share a screenshot of what happened with `installation` as folder name?

Comment (sch00lb0y) [inline README.md]: Thanks for noticing. there is no such problem. please check the preview http://yaydoc.herokuapp.com/preview/rbalajis25@gmail.com/11e8c332-6e57-49f6-8821-88fe7c948e9a_preview/

File: docs/general/roles.md [hunk 2]
```diff
@@ -19,13 +17,13 @@ There are three System Roles:
...
 The Super Admin can access all user features.
 
-| Function | Role | View | Create, Edit, Assign, Unassign | Delete | BaseURLs |
-| --- | --- | --- | --- | --- | --- |
-| MANAGE OWN PROFILE | Super Admin | YES | YES | YES | /profile
-| MANAGE OWN EVENTS | Super Admin | YES | YES | YES | /create, /events, /events/ID
-| IMPORT/EXPORT OWN EVENTS | Super Admin | YES | YES | YES |
-| MANAGE OWN SESSIONS | Super Admin | YES | YES | YES |
-| MANAGE SESSIONS OF OWN EVENTS | Super Admin | YES | YES | YES |
+Function | Role | View | Create, Edit, Assign, Unassign | Delete | BaseURLs
```
Comment (pri22296) [inline docs/general/roles.md]: Is this change necessary?

Comment (imujjwal96) [inline docs/general/roles.md]: instead add `|` at the end of those missing and check for those rows with less than 6 columns

Comment (sch00lb0y) [inline docs/general/roles.md]: updated
